libvirt/acceptance_tests_functions_helpers.go

```diff
@@ -21,5 +27,59 @@ func getResourceFromTerraformState(resourceName string, state *terraform.State)
 		return nil, fmt.Errorf("No libvirt resource key ID is set")
 	}
 	return rs, nil
+}
+
+// test in all testacc that resource is destroyed
+func testaccCheckLibvirtDestroyResource(resourceName string, virConn libvirt.Connect) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		for _, rs := range s.RootModule().Resources {
+			if rs.Type != resourceName {
+				continue
+			}
+
+			_, err := virConn.LookupDomainByUUIDString(rs.Primary.ID)
```

Comment:
Yop, true i overlooked the call for network. So the function is not generic.. and is buggy :jack_o_lantern:  i will add a checkand adapt it for network res

Reply:
I open an issue as reminder for this

Reply:
It is not only for network. The volume one also does a different call, and the cloudinit one is more or less similar to the volume one, only because cloudinit is implemented using volumes.